examples/README.md

```diff
@@ -4,7 +4,16 @@ This directory serves as a demo of how you can develop and run individual compon
 
 ## Getting started
 
-The easiest way of running these demos is using a Docker container. Type `make help` to get help for the different make targets to build, run and stop the container. Once the docker container is running, you can connect to the Jupyter notebook server by copying and pasting the correct address into any browser on your machine.
+The easiest way of running these demos is using a Docker container. If you do not already have Docker installed, it can be downloaded directly from [Docker](https://www.docker.com/). Once Docker is set up, you can build and run the container:
+
+```
+make build
+make run
+```
+
+Once the docker container is running, you can connect to the Jupyter notebook server by copying and pasting the URLs into any browser on your machine. If you would like to retain the changes made to the notebooks, you can mount the local notebooks folder to the container. Instead of `make run`, do `make dev`.
```

Comment:
```suggestion
Once the docker container is running, you can connect to the Jupyter notebook server by copying and pasting the URLs into any browser on your machine.
If you would like to retain the changes made to the notebooks, you can mount the local notebooks folder into the container by running `make dev` instead of `make run`.
```